tilequeue/metatile.py

```diff
@@ -0,0 +1,51 @@
+import zipfile
+import StringIO
+from collections import defaultdict
+from tilequeue.format import zip_format
+
+
+def make_single_metatile(size, tiles):
+    """
+    Make a single metatile from a list of tiles all having the same
+    coordinate and layer.
+    """
+
+    assert size == 1, \
+        "Tilequeue only supports metatiles of size one at the moment."
+
+    if len(tiles) == 0:
+        return []
+
+    coord = tiles[0]['coord']
+    layer = tiles[0]['layer']
+
+    buf = StringIO.StringIO()
+    with zipfile.ZipFile(buf, mode='w') as z:
+        for tile in tiles:
+            assert tile['coord'] == coord
+            assert tile['layer'] == layer
+
+            tile_name = '0/0/0.%s' % tile['format'].extension
```

Comment:
> I couldn't see any downsides to making it relative, but I could easily be overlooking something. What do you think?

Other than the obvious that consumers need to translate between the two, I can't think of any downsides either. And like you mentioned, relative coordinates could allow for more flexibility. :+1: